Classes/Domain/Service/ZoomApiService.php

```diff
@@ -110,29 +107,14 @@ public function getUpcomingMeetings(bool $skipCache = false): array
      */
     public function getRecordings(DateTime|string $from, DateTime|string $to, bool $skipCache = false): array
     {
-        $cacheEntryIdentifier = sprintf('recordings_%s_%s', is_string($from) ? $from : $from->format('Y-m-d'), is_string($to) ? $to : $to->format('Y-m-d'));
+        list($from, $to) = $this->convertFromAndToAsDateTimeImmutable($from, $to);
```

Comment:
I think we should stick with the array short syntax (here and in `convertFromAndToAsDateTimeImmutable`)
```suggestion
        [$from, $to] = $this->convertFromAndToAsDateTimeImmutable($from, $to);
```